lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/DoiUtilsTest.java

```diff
@@ -47,27 +41,27 @@ public void testConvertToDto() {
 		assertEquals(updatedOn.getTime(), dto.getUpdatedOn().getTime());
 	}
```

Comment:
add test for object that is not Entity with dbo->dto and dto->dbo (no syn on the ids)